astor/rtrip.py

```diff
@@ -85,6 +85,10 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s, pre_encoded=(type(b'') is type(''))):
```

Comment:
That would certainly do it.  It's not my preference, but I've come to understand that my extreme views on duck typing (look for whether something will work or not at the lowest possible level) are not widely shared, so I'll make that change.